src/scale_audio_rating.py

```diff
@@ -292,7 +312,8 @@ async def main(cfg, args):
                     ],
                 },
             ],
-            "metadata": metadata
+            "metadata": metadata,
+            "tags":[file_urls, args.project]
         }
         task_obj['metadata']["group"] = args.project
 

```

Comment:
Should we add finalizing the batch to the script as well?

Reply:
oh yes! good call

Reply:
added